django_lifecycle/mixins.py

```diff
@@ -251,7 +252,9 @@ def _watched_fk_model_fields(cls) -> List[str]:
     def _watched_fk_models(cls) -> List[str]:
         return [_.split(".")[0] for _ in cls._watched_fk_model_fields()]
 
-    def _get_hooked_methods(self, hook: str, **kwargs) -> List[AbstractHookedMethod]:
+    def _get_hooked_methods(
+        self, hook: str, update_fields: Iterable[str] | None = None, **kwargs
+    ) -> List[AbstractHookedMethod]:
```

Comment:
if you are import annotations from `__future__` why not
```suggestion
    ) -> list[AbstractHookedMethod]:
```
?

Reply:
Thought it wasn't available in py3.7. Will do a refactor later, thanks!